scpd/lms/static/sass/_overrides.scss

```diff
@@ -0,0 +1,410 @@
+/****************************************************************************************************
+*   Stanford Overrides File
+*
+*   Contains all the CSS overrides for the Stanford Open Source MOOC Project
+*
+****************************************************************************************************/
+
+//-----------------
+//  Mixin Styles
+//-----------------
+@mixin login_register_h1_style {
+    clip: auto !important;
+    margin-top: 25px;
+    width: 750px;
+    top: 0px !important;
+}
+
+@mixin footer_references_style {
+    margin: 0 !important;
+    text-align: right;
+}
+
+.wrapper-footer {
+    padding: 10px 10px 0 10px !important;
+}
+
+.view-register, .view-login, .view-passwordreset {
+    .introduction header .title {
+        top: 20px;
+        left: 20px;
+    }
+}
+
+//-----------------------
+//  Stanford Extra Styles
+//-----------------------
+#stanford-footer-edx-image {
+    display: block;
+    height: 40px;
+    width: 120px;
+    padding-bottom: 10px;
+    float: right;
+    background-image: url("https://files.edx.org/openedx-logos/edx-openedx-logo-tag.png");
```

Comment:
eh, let's stop linking off-site and just load from our own assets;
this can slow down page loads.